wiki/Coding-Standards.md

```diff
@@ -0,0 +1,99 @@
+# Coding Standards
+
+In this repository we enforce the following coding standards.
+
+## Tools
+### ESLint
+In this project we use [EsLint](https://eslint.org/) as Linter and use the following configuration:
+
+```json    
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": [
+    "plugin:react/recommended",
+    "airbnb",
+    "prettier",
+    "prettier/react"
+  ],
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 12,
+    "sourceType": "module"
+  },
+  "plugins": ["react", "only-warn"],
+  "rules": {
+    "no-console": ["error", { "allow": ["warn", "error"] }]
+  },
+  "root": true
+}
+
+```
+
+[Here](https://eslint.org/docs/rules/) you can find the rules that eslint enforces.
+### Prettier
+Another Code Formatting Tool being used is [Prettier](https://prettier.io/). Here we use the following configuration:
+
+```json
+{
+  "tabWidth": 2,
+  "bracketSpacing": true,
+  "endOfLine": "lf",
+  "jsxSingleQuote": true,
+  "semi": true,
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "printWidth": 80,
+  "useTabs": false
+}
+
+```
+
+[Here](https://prettier.io/docs/en/options.html) you can find all the rules that prettier enforces by default.
+
+**These tools enforce a lot of formatting, code-quality and simplifications that give use a base layer of standards. Please install these in your IDE and configure them as stated above.**
+
+## Coding Standards
+
+### Naming
+- Variable Names in `lowerCamelCase`
+- "Single Source of Truth" is abbreviated with `Ssot` or `ssot` (don't use `SSoT` or `SSOT`)
+- use `Robot` in all cases instead of `Bot`
+- use `Application` in all cases instead of `App` (in context of supported RPA Applications)
```

Comment:
Here I would add:
- use PascalCase for the naming of components (.jsx files)
- use camelCase for the naming of all other files (.js files mainly)
-  use UPPERCASE for constants